charts/barbican/charts/helm-toolkit/Chart.yaml

```diff
@@ -9,4 +9,4 @@ name: helm-toolkit
 sources:
 - https://opendev.org/openstack/openstack-helm-infra
 - https://opendev.org/openstack/openstack-helm
-version: 0.2.55
```

Comment:
Please pin helm-toolkit to 0.2.55 as we've done in the other charts in order to avoid a this big commit change and make it a much smaller change.